tests/views_integration/Tribe/Events/Views/V2/HTML/Month/MonthEventMultidayTest.php

```diff
@@ -9,7 +9,20 @@ class MonthEventMultidayTest extends TestHtmlCase {
 	 * @test
 	 */
 	public function it_should_contain_correct_html_classes() {
-		$template = $this->template->template( 'month/event-multiday', [ 'event' => (object) [ 'ID' => 0 ] ] );
+
+		$event = [
```

Comment:
Why not use the Event factory here? See https://github.com/moderntribe/the-events-calendar/blob/9ef75801268a195c91f0525d13f1693f9c305288/tests/_support/Factories/Event.php#L7 and https://github.com/moderntribe/the-events-calendar/blob/7b736aadc603b10bbb8a46a6ac12514882edd8e3/tests/wpunit/Tribe/Events/REST/V1/Endpoints/Single_Organizer_SlugTest.php#L30

Reply:
Yep, I was just setting the smallest piece of data that the template will require so the test doesn't die. And the idea was coming back once the templates actually have real data an adapt the tests. (Basically, my idea of adding the tests at this point is that will make us come back and fix them later, and we are sure we'll have tests for that template, but at this point there's no data or anything to test with).

Reply:
And also, at this point we're not actually using an `$event` per se, but a simple small implementation of an object with a few pieces of info to display the demo. Will leave a `@todo`